pkg/rpc/rpctest/rpctest.go

```diff
@@ -0,0 +1,17 @@
+package rpctest
+
+import (
+	"net/http/httptest"
+	"strings"
+
+	"github.com/testground/testground/pkg/rpc"
+)
+
+// NewRecordedOutputWriter returns an OutputWriter where the response is recorded.
+func NewRecordedOutputWriter(req_id string) (rec *httptest.ResponseRecorder, ow *rpc.OutputWriter) {
+	req := httptest.NewRequest("GET", "/", strings.NewReader(""))
+	req.Header.Add("X-Request-ID", req_id)
+	rec = httptest.NewRecorder()
+	ow = rpc.NewOutputWriter(rec, req)
+	return rec, ow
```

Comment:
@coryschwartz ^^ still needs fixing. (golang.org/doc/effective_go.html#mixed-caps)

```suggestion
func NewRecordedOutputWriter(reqID string) (rec *httptest.ResponseRecorder, ow *rpc.OutputWriter) {
	req := httptest.NewRequest("GET", "/", strings.NewReader(""))
	req.Header.Add("X-Request-ID", reqID)
	rec = httptest.NewRecorder()
	ow = rpc.NewOutputWriter(rec, req)
	return rec, ow
```